### PR TITLE
Init Arabic Localization

### PR DESCRIPTION
**Pull request by mohamedsgap**
Redone K8S Arabic Localization on this PR, because an error has occurred on my previous PR

### CODE REVIEW

**Comment by mohamedsgap** [comment]
Hey @sftim, idk what's wrong again with the new branch and PR ??

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -233,3 +233,9 @@ aliases:
     - idvoretskyi
     - MaxymVlasov
     - Potapy4
+  sig-docs-ar-owners: # Admins for Ukrainian content
+    - mohamedsgap
+    - mustafaAnwar2020
+  sig-docs-ar-reviews: # PR reviews for Ukrainian content
+    - mohamedsgap
+    - mustafaAnwar2020
```
Comment (sftim) [inline OWNERS_ALIASES:241]: Ukrainian?

**Comment by sftim** [comment]
Does this branch preview locally @mohamedsgap ?

If you can't run the local preview, look at GitHub's [list of checks](https://github.com/kubernetes/website/pull/20343/checks) and view the logs for the ones that failed?

File: OWNERS_ALIASES (hunk 1 above)
Comment (mohamedsgap) [reply OWNERS_ALIASES]: oh sorry i miss this typo!

**Comment by mohamedsgap** [comment]
hey, mate @mhamdisemah, I've just add you and the owner with me, we need to fix this issue that checks were not pass, we gonna be ready to merge this PR, and build on top of it

**Comment by sftim** [comment]
@mhamdisemah / @mohamedsgap would you like any help troubleshooting this PR?

**Comment by sftim** [comment]
@mohamedsgap the localization needs to progress to become mergeable and adopted
Without that progress I think it will make sense to close the PR

Is there any help you'd like to move things forward?

**Comment by mohamedsgap** [comment]
@sftim we are making progress on @mhamdisemah's PR #20459,  because it's much cleaner and pass the CI, we only have one page left to be able to merge it.

**Pull request closed without merging**